dicom-archive/DICOM/DCMSUM.pm

```diff
@@ -119,13 +121,14 @@ QUERY
 	    print "\n\nPROBLEM:\n The user \'$row[3]\' has already inserted this study. \n The unique study ID is $row[0]\n";
 	    print " This is the information retained from the first time the study was inserted:\n $row[1]\n\n";
 	    print " Last update of record :\n $row[2]\n\n";
-	    exit 33;
+	    exit $NeuroDB::ExitCodes::FILE_NOT_UNIQUE;
 	}
 	# do not allow to run diccomSummary with database option if the study has already been archived
 	elsif (!$Archivemd5 && $row[3] ne "") { 
 	    print "\n\n PROBLEM: This study has already been archived. You can only re-archive it using dicomTar!\n";
 	    print " This is the information retained from the first time the study has been archived:\n $row[1]\n\n";
-	    exit 33; }
+	    exit $NeuroDB::ExitCodes::FILE_NOT_UNIQUE;
+        ; }
```

Comment:
extra set of `;` here
also not sure why the error is File_Not_Unique; is that really what is happening here?

Reply:
good catch! Correcting the duplicated ;

For the exit code, that is what I undestand. The study archive (or file) has already been inserted into the database so is not unique (like on the tarchiveloader side with MINC files). Does that make sense?